samples/openthread/cli/sample.yaml

```diff
@@ -14,25 +15,32 @@ tests:
       - nrf21540dk_nrf52840
   samples.openthread.cli.thread_1_1:
     build_only: true
-    platform_allow: nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840 nrf52833dk_nrf52833 nrf21540dk_nrf52840
+    platform_allow: nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840 nrf21540dk_nrf52840
```

Comment:
Is there a memory issue with the 1.1 build? I believe the default 1.2 build should be modified instead.